lambda/app/src/controllers/user.ts

```diff
@@ -142,19 +145,50 @@ export const isAllowedToManageRoles = async (session: Session, integrationId: nu
   return canCreateOrDeleteRoles(integration);
 };
 
-export const deleteStaleUsers = async (userId: string) => {
+export const deleteStaleUsers = async (user: any) => {
   try {
-    const existingUser = await models.user.findOne({ where: { idir_userid: userId } });
+    if (user.clientData && user.clientData.length > 0) {
+      user.clientData.map(async (cl: { client: string; roles: string[] }) => {
+        const integration = await models.request.findOne({
+          where: {
+            clientId: cl.client,
+          },
+          raw: true,
+        });
+
+        if (integration.teamId) {
+          const userEmails = await getAllEmailsOfTeam(integration.teamId);
+          let isTeamAdmin = false;
+          userEmails.map((u: any) => {
+            if (u.idir_email === user.email && u.role === 'admin') {
+              isTeamAdmin = true;
+            }
+          });
+          sendTemplate(EMAILS.DELETE_INACTIVE_IDIR_USER, {
+            teamId: integration.teamId,
+            username: user.attributes.idir_username || user.username,
+            clientId: cl.client,
+            roles: cl.roles,
+            teamAdmin: isTeamAdmin,
```

Comment:
so this sends these emails to everyone, not just the  admin?

Reply:
yes to all the team members